src/platform/site-wide/sass/modules/_m-homepage.scss

```diff
@@ -95,6 +95,7 @@
     color: $color-white;
     text-decoration: none;
     background-color: $color-primary;
+    outline-offset: 2px;
```

Comment:
Honestly never seen this property before.

Reply:
It's only used once in the codebase, in [`merger.scss`](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/site-wide/sass/merger.scss#L618). Something tells me it's defaulting differently based on the element type, because there doesn't seem to be any explicit overrides/resets (that I can tell).